omx/gstomxaacdec.c

```diff
@@ -232,6 +275,7 @@ gst_omx_aac_dec_is_format_change (GstOMXAudioDec * dec, GstOMXPort * port,
 static gint
 gst_omx_aac_dec_get_samples_per_frame (GstOMXAudioDec * dec, GstOMXPort * port)
 {
+  g_return_val_if_fail (dec, NULL);
```

Comment:
Returning `NULL` in a `gint` method

Reply:
Ready for review: 0947ed4a695b58965705a115cfebaebbfdd2f0db